src/components/select-field/SelectFieldDropdown.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+
+import type { SelectOptionValueProp } from './props';
+import type { PopperChildrenProps } from '../popper/props';
+
+export const OVERLAY_SCROLLABLE_CLASS = 'bdl-SelectField-overlay--scrollable';
+
+type Props = {
+    children: React.Node,
+    innerRef?: React.Ref<any>,
```

Comment:
Should this be UL element? Instead of any

Reply:
`HTMLUListElement` is incompatible with `React.Component` so it fails the flow-type.